### DISCUSSION TITLE
Copilot4Eclipse Chat Scrolls down automatically when generating response

### DISCUSSION THREAD

**Discussion by arturaswm2** [Genuitec/Copilot4Eclipse-community#35] [Help]
1. Please describe the issue you are experiencing. Include as much detail and relevant context and timing information as possible. Since we can not see your screen we rely on you to be our eyes into the problem. Avoid vague, ambiguous or off-topic problem descriptions.

When you use Copilot4Eclipse Chat functionality in Eclipse, when you write a prompt and start receiving the answer, the answer by the current nature of genAI based chatbots is that it starts generating live, therefore you start reading in the beginning, but the problem is, it does not stay in the beginning, it scrolls down, which is extremely annoying and you scroll up and then it scrolls down again and again and you're basically unable to read until it has finished generatic because it keeps scrolling it down by force as text appears.

1a. Describe the steps to reproduce if possible. Include any timing information if relevant.

Just type any prompt and as soon as the window gets full, it starts scrolling down to show the newly generated text. HAPPENS ALWAYS.

2. Details about your Eclipse environment? You can find this information at Help menu > About > Installation Details button > Configuration tab.

Minimally we need the following information:

Eclipse IDE for Enterprise Java and Web Developers (includes Incubating components)
Version: 2024-06 (4.32.0)
Build id: 20240606-1231
OS: Windows 11, v.10.0, x86_64 / win32
Java vendor: Eclipse Adoptium
Java runtime version: 21.0.4+7-LTS
Java version: 21.0.4

4. Provide any additional observations and information that you feel will help us quickly identify the issue.

Seems like a critical bug - it's pretty much impossible to use it without cursing after every single prompt as it happens. If that can't be resolved then it's easier to use ChatGPT or any other external GenAI assistant because they don't have this problem.

**Comment by wayneparrott** [comment]
Thx for taking time to share this constructive feedback. 
I've opened a team discussion and issue with your comments. We will give this serious consideration. 
Our window for updates is locked down atm for the upcoming 1.3.0 update in the next 1-2 weeks. Following that update we plan to include UX improvements in the following update scope.

Wayne
Copilot4Eclipse Team

**Reply by arturaswm2** [reply]
Sounds great! Thank you very much guys! Yeah, if it happens to many or most people then it's really quite an important thing, because many people come and wanna try genAI assisted development, especially with IDE integration and if they encounter annoying things they might just think that the current state isn't ready yet and leave it for multiple years and tell their colleagues or supervisors that they tried it and it wasn't very good. But yeah, other than that - keep up the good work und good luck!
